### ISSUE TITLE
[Bug]: Vibrancy works for a second after launch and then goes away

### ISSUE THREAD

**Issue by nathan-fiscaletti** [illixion/vscode-vibrancy-continued#141]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I'm experiencing an issue where I see the vibrancy effect for a split second after launching VSCode but it then immediately goes away.

I recorded a [short video](https://www.youtube.com/watch?v=Sjc64HecHQY) demonstrating this behavior.

### Expected Behavior

The vibrancy effect to stay in place.

### Steps To Reproduce

1. Select Dark+ (default dark) theme.
2. Install extension
3. Run "Reload Vibrancy"
4. Restart VSCode

### Environment

```markdown
- OS: Windows 11 (Version 10.0.22621.3296)
- VSCode: 1.87.2
- Extension: 1.1.29
- Theme: Dark+ (default dark)
```

**Comment by illixion** [comment]
Do you have any other extensions that modify internal VSCode files, like APC or VSCode animations? Also, please try reinstalling VSCode, this will help ensure that your installation is consistent and you won't lose any settings.

**Comment by nathan-fiscaletti** [comment]
Here is a list of all of the extensions I have installed.

```
42crunch.vscode-openapi
aaron-bond.better-comments
bierner.markdown-mermaid
cweijan.dbclient-jdbc
cweijan.vscode-database-client2
dart-code.dart-code
dbaeumer.vscode-eslint
dionmunk.vscode-notes
eamodio.gitlens
edgaraldair.vintage-vibrance
esbenp.prettier-vscode
formulahendry.code-runner
formulahendry.docker-explorer
george-alisson.html-preview-vscode
github.copilot
github.copilot-chat
github.vscode-pull-request-github
gitlab.gitlab-workflow
golang.go
goopware.raythis
hediet.vscode-drawio
icrawl.discord-vscode
italypaleale.go-playground
jock.svg
joselitofilho.ginkgotestexplorer
kumar-harsh.graphql-for-vscode
malcomnm.fox
marp-team.marp-vscode
mechatroner.rainbow-csv
ms-azuretools.vscode-docker
ms-dotnettools.csharp
ms-dotnettools.vscode-dotnet-runtime
ms-python.black-formatter
ms-python.debugpy
ms-python.isort
ms-python.python
ms-python.vscode-pylance
ms-toolsai.jupyter
ms-toolsai.jupyter-keymap
ms-toolsai.jupyter-renderers
ms-toolsai.vscode-jupyter-cell-tags
ms-toolsai.vscode-jupyter-slideshow
ms-vscode-remote.remote-wsl
ms-vscode.cmake-tools
ms-vscode.cpptools
ms-vscode.cpptools-extension-pack
ms-vscode.cpptools-themes
ms-vscode.live-server
ms-vsliveshare.vsliveshare
mtxr.sqltools
mtxr.sqltools-driver-pg
nathanfiscaletti.go-play
osohq.oso
p1c2u.docker-compose
pnp.polacode
postman.postman-for-vscode
redhat.java
redhat.vscode-yaml
shyykoserhiy.vscode-spotify
streetsidesoftware.code-spell-checker
svelte.svelte-vscode
tobiah.unity-tools
tomoki1207.pdf
twxs.cmake
visualstudioexptteam.intellicode-api-usage-examples
visualstudioexptteam.vscodeintellicode
vs-publisher-1448185.keyoti-changeallendoflinesequence
vscjava.vscode-java-debug
vscjava.vscode-java-dependency
vscjava.vscode-java-pack
vscjava.vscode-java-test
vscjava.vscode-maven
vscode-icons-team.vscode-icons
wakatime.vscode-wakatime
wallabyjs.console-ninja
yile-ou.paddy-color-theme
zhuangtongfa.material-theme
```

**Comment by TomDevX** [comment]
I have the same issue as you. Moreover, my extension always turns my background to light themes. For now, I still don't know how I can fix it.

**Comment by FigueiredoStable** [comment]
> I have the same issue as you. Moreover, my extension always turns my background to light themes. For now, I still don't know how I can fix it.

The same with me, on MacOs Sonoma i try to reinstall vscode to, without any others extensions.

**Comment by irondude14** [comment]
did anyone managed to fix it? have exactly the same issue

**Comment by devCKVargas** [comment]
> did anyone manage to fix it? have exactly the same issue

It's now working after I disabled APC extension (it conflicts) and switched to Pre-Release Version. Even after returning to Release Version, vibrancy is now working.

Now I'm waiting for PR #134 to be merged.

**Comment by illixion** [comment]
If anyone is still experiencing this issue, please reply to this thread with info about your OS version, VSCode version, extension version and the theme you're using. Additionally, ensure that you don't have transparency effects disabled globally through your OS settings. This can usually be found under Accessibility settings, and it may be called "Transparency effects" or "Reduce transparency."

> Now I'm waiting for PR #134 to be merged.

It is now merged and the fix is available in v1.1.34

**Comment by irondude14** [comment]
> If anyone is still experiencing this issue, please reply to this thread with info about your OS version, VSCode version, extension version and the theme you're using. Additionally, ensure that you don't have transparency effects disabled globally through your OS settings. This can usually be found under Accessibility settings, and it may be called "Transparency effects" or "Reduce transparency."
> 
> > Now I'm waiting for PR #134 to be merged.
> 
> It is now merged and the fix is available in v1.1.34

I found the core of the problem. In Settings > Extensions > Vibrancy Effect, my "Vscode_vibrancy: Theme" was set to "Default Dark" instead of "Dark+". As soon as I changed it to "Dark+" and restarted, the effect started working.

**Comment by TomDevX** [comment]
> > If anyone is still experiencing this issue, please reply to this thread with info about your OS version, VSCode version, extension version and the theme you're using. Additionally, ensure that you don't have transparency effects disabled globally through your OS settings. This can usually be found under Accessibility settings, and it may be called "Transparency effects" or "Reduce transparency."
> > > Now I'm waiting for PR #134 to be merged.
> > 
> > 
> > It is now merged and the fix is available in v1.1.34
> 
> I found the core of the problem. In Settings > Extensions > Vibrancy Effect, my "Vscode_vibrancy: Theme" was set to "Default Dark" instead of "Dark+". As soon as I changed it to "Dark+" and restarted, the effect started working.

I don't even have the "Dark+" option there.

**Comment by illixion** [comment]
Correct, there is no such option in the current version of Vibrancy. Perhaps they meant that they've changed their VSCode theme to Dark+. If anyone experiencing this issue is willing to provide the info I've requested, then I could try to help.

<img width="366" alt="Screenshot 2024-06-12 at 09 24 34" src="https://github.com/illixion/vscode-vibrancy-continued/assets/14954655/9d15194f-07a8-424b-9cfd-ea6c87e65991">

**Comment by irondude14** [comment]
I meant "Solarized Dark+", thats the only "Dark+" theme available
